src/components/Buttons/Button.tsx

```diff
@@ -55,6 +57,7 @@ export const Button: React.FC<ButtonProps> = ({
 
   return (
     <button
+      data-cy={dataCy || "button"}
```

Comment:
Consider if it is a good idea to have a default value for `cy`.

I do not think it provides much value to select elements using `getBySel('button')` instead of the standard `get('button')`  In other situations we make the argument optional.

Using default values also occurs other places throughout this PR ranging from the specific to the very generic. This seems to be the most generic option.

Reply:
You are right. I think this solution was a crossover with a change from Sine and me. So I'll now remove the default value with the danger of another test starts failing :)